### ISSUE TITLE
Calculated run time always 2,5h

### ISSUE THREAD

**Issue by tomseii** [kotope/ha_car_heater#1]
No matter what time i input, the calculated runtime is always 2,5?

**Comment by kotope** [comment]
Runtime is calculated from outside temperature.
So most probably the temperature is less than -25 degrees celsius or the sensor cannot be read correctly.

I suggest verifying the temperature sensor value or weather service (if in use).

**Comment by wlfrdssn** [comment]
I got the same problem as above. I have tried to use my own temperature sensor and  SMHI weather service with no result. Its just say 2.5 hour all the time. Im not that familiar to automations yet. This is what I have tried (I want to use my own temperature sensor)

states.sensor.out_temp
sensor.out_temp
states.weather.smhi_hemma.attribtutes.temperature

![image](https://user-images.githubusercontent.com/61554783/149298528-56bf4914-fab2-4b91-abf5-ee7533868060.png)
